src/Worker/UpdateServerPeers.php

```diff
@@ -57,13 +57,13 @@ public static function execute(string $url)
 		$total = 0;
 		$added = 0;
 		foreach ($peers as $peer) {
-			if (Network::isUrlBlocked('http://' . $peer)) {
+			if (Network::isUrlBlocked('https://' . $peer)) {
```

Comment:
We only match on the domain so far, so this change would be inconsequential, although it looks better?

Reply:
Okay, then. http:// isn't part of the validation.

Reply:
Yes, but we still need a scheme to be able to take advantage of the URL parsing method used in `Network::isUrlBlocked`